docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,64 @@
+# Deploy the GitHub runner image builder for the first time
+
+## What you'll do
+
+- Deploy the charm.
+- Integrate with GitHub runners.
+
+## Requirements
+
+- A working station, e.g., a laptop, with amd64 architecture.
+- Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by 
```

Comment:
Does this tutorial really need `a MicroK8s controller`?

Reply:
Also do we need to bootstrap the OpenStack controller? If yes we gotta show how to.
I am not sure if Multipass comes with OpenStack to be honest. If we need to install OpenStack just like we would do in local it might be unnecessary to suggest using Multipass. @erinecon wdyt?

Reply:
There is a step underneath to help bootstrap OpenStack using the Canonical MicroStack. This can work with an LXD controller for local testing.

Reply:
I've updated the tutorial so that we use Multipass w/ OpenStack and Juju installed on the same VM - I can't test this step due to the limitations of my workstations (laptop, server). I don't think anyone can quite test this step within the team without a dedicated machine.